Temporary_PHPCompatibility_Test.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+// Should tell me return void is not available in PHP 5.6
+class Foo {
+	public function do_nothing(): void {
+
+	}
+}
+
+// Should tell me anonymous class are not available in PHP 5.6
+$a = new class {}
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: f0e90559592e54bbe82db68036616ab0</sub>